build/pmm-client-docker/Dockerfile

```diff
@@ -1,20 +1,25 @@
-FROM centos:7
+FROM registry.access.redhat.com/ubi7/ubi
 RUN yum -y install iproute nc \
     && rm -rf /var/cache/yum
 
 COPY pmm2-client.tar.gz /tmp/pmm2-client.tar.gz
+RUN groupadd -g 1002 pmm-agent
+RUN useradd -u 1002 -r -g pmm-agent -s /sbin/nologin \
+            -d /usr/local/percona/pmm2 \
+            -c "Default Application User" pmm-agent
+
+RUN install -d -o pmm-agent -g pmm-agent -m 0775 /usr/local/percona/pmm2 \
+    && install -d -o pmm-agent -g pmm-agent -m 0775 /run /var/log /etc/rc.d/init.d
+
 RUN tar -zxpf /tmp/pmm2-client.tar.gz -C /tmp \
     && cd /tmp/pmm2-client-* \
+    && sed -i 's:install:install -o pmm-agent -g pmm-agent:g' ./install_tarball \
```

Comment:
Let's avoid adding more `sed`s where possible: it is too brittle, and breaks from basic refactorings, renamings, and reformattings.
It is entirely possible to avoid `sed` there by adding parameters to `install_tarball` script: accept optional `PMM_INSTALL_USER` and `PMM_INSTALL_GROUP` environment variables, add them to `install` commands if they are not empty.

Reply:
That is, _if_ we actually need those permissions.

Reply:
done